big-int/src/operators/greater_than.cpp

```diff
@@ -32,5 +32,40 @@
 
 namespace libbig
 {
+bool largeInt::operator>(const largeInt &a)
+{
+    if(this->sign == NEGATIVE && a.sign == POSITIVE)
+    {
+        return 0;
+    }
+    if(a.sign == NEGATIVE && this->sign == POSITIVE)
+    {
+        return 1;
+    }
+    if(this->number.length() > a.number.length())
+    {
+        return 0^this->sign;
+    }
+    if(this->number.length() < a.number.length())
+    {
+        return 1^this->sign;
+    }
 
+    for(int i=0;i<a.number.length();i++)
+    {
+        if(this->number[i]==a.number[i])    
```

Comment:
you can remove this condition, it is redundant

Reply:
It is not redundant because if I remove this I need to add another condition to check if `this->number[i]<a.number[i]`